hack/jenkins/common.sh

```diff
@@ -151,10 +153,11 @@ if [[ "${VM_DRIVER}" == "hyperkit" ]]; then
   fi
 fi
 
-if pgrep kubectl; then
-  echo "killing hung kubectl processes ..."
-  ps -afe | grep kubectl | grep -v grep || true
-  pgrep kubectl | xargs kill || true
+kprocs=$(pgrep kubectl || true)
+if [[ "${kprocs}" != "" ]]; then
+  echo "error: killing hung kubectl processes ..."
+  ps -f -p ${kprocs} || true
+  kill ${kprocs} || true
```

Comment:
why not -9?

Reply:
There isn't a benefit. Extra kubectl's won't interfere other than driving the process count up.